extensions/amp-fit-text/0.2/fit-text.css.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const LINE_HEIGHT_EM_ = 1.15;
```

Comment:
Is this really the old default? Very non-standard...

Reply:
[Yes, here.](https://github.com/ampproject/amphtml/blob/master/extensions/amp-fit-text/0.1/amp-fit-text.js#L22) Anything we can do about it you think?

Reply:
Initially, it's very important to make it overridable. Everything else is blocked by the same stylesheet issues. So let's ensure the first aspect and we'll see.

Reply:
A trouble with making this configurable is that line-height takes a few different units, and the component uses it to calculate the max # of lines for clamping. The way I see it the options are -- to take `em` ints as a prop or take any string as a prop or style prop, but we have to parse for units and special case the calculations for clamping.

Reply:
We could alternatively break away from `em` units and take unitless ints as a prop, but this goes into the gray area of how much we should be changing from v1. According to the spec the unitless number is preferred to `em` since `em` can be unpredictable. Coincidentally the equation wouldn't change if we do this.

Reply:
Ok, let's default for now as we do and add a TODO to https://github.com/ampproject/amphtml/issues/28281. I'll get the info on what our best options are.